### ISSUE TITLE
Custom select not working with screenreader

### ISSUE THREAD

**Issue by thijsvdanker** [livewire/flux#1566]
### Flux version

v2.1.6

### Livewire version

v3.6.3

### Tailwind version

v4.1.5

### Browser and Operating System

Firefox on MacOS

### What is the problem?

When using the custom select (https://fluxui.dev/components/select#custom-select) with VoiceOver it does not announce all the options.

The combobox (https://fluxui.dev/components/select#combobox) does work as expected.

### Code snippets

<h2>Listbox from custom select example: not working with VoiceOver</h2>

```
<flux:select variant="listbox" placeholder="Choose industry...">
    <flux:select.option>Photography</flux:select.option>
    <flux:select.option>Design services</flux:select.option>
    <flux:select.option>Web development</flux:select.option>
    <flux:select.option>Accounting</flux:select.option>
    <flux:select.option>Legal services</flux:select.option>
    <flux:select.option>Consulting</flux:select.option>
    <flux:select.option>Other</flux:select.option>
</flux:select>
```

<h2>Combobox from combobox example: working with VoiceOver</h2>

```
<flux:select variant="combobox" placeholder="Choose industry...">
    <flux:select.option>Photography</flux:select.option>
    <flux:select.option>Design services</flux:select.option>
    <flux:select.option>Web development</flux:select.option>
    <flux:select.option>Accounting</flux:select.option>
    <flux:select.option>Legal services</flux:select.option>
    <flux:select.option>Consulting</flux:select.option>
    <flux:select.option>Other</flux:select.option>
</flux:select>
```

### How do you expect it to work?

When I have focus on the [custom select](https://fluxui.dev/components/select#custom-select) and use arrow-down to navigate the options, VoiceOver should announce the active option.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.
